src/components/sidebar.tsx

```diff
@@ -209,6 +214,28 @@ export function Sidebar({ className }: SidebarProps) {
             </Button>
           ))}
         </div>
+
+        <SidebarCategory>Site</SidebarCategory>
```

Comment:
I think this and settings fit the Misc category, and what's in misc should should probably be under collections, considering they're literally tracking collections of things lmao.

Reply:
good catch